### DISCUSSION TITLE
版本管理混乱问题

### DISCUSSION THREAD

**Discussion by zhengjiaao** [alibaba/jetcache#889] [Ideas]
感觉集成到springboot难度挺高的，jetcache版本没有与springboot对应把

**Comment by areyouok** [comment]
同一个jetcache的版本可以兼容spring/spring-boot多个版本，都是测试过的，具体范围见兼容性文档：

https://github.com/alibaba/jetcache/blob/master/docs/CN/Compatibility.md

因为spring的版本会有不兼容的修改，所以要同时适配多个spring版本是有一些麻烦的。更麻烦的是jedis这东西每次升级都有很大的不兼容，那我也没有好办法了。

如果遇到兼容性问题，自己解决。解决以后如果发现是jetcache的问题导致的，可以提一个PR过来。

**Reply by zhengjiaao** [reply]
收到，感谢，问题已经找到，是配置项导致问题，开始以为是依赖问题，具体详见：https://github.com/alibaba/jetcache/issues/890
